Cookbook/Cookbook.xcodeproj/project.pbxproj

```diff
@@ -321,7 +325,7 @@
 				ASSETCATALOG_COMPILER_GLOBAL_ACCENT_COLOR_NAME = AccentColor;
 				CODE_SIGN_ENTITLEMENTS = Cookbook/Cookbook.entitlements;
 				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = 9W69ZP8S5F;
+				DEVELOPMENT_TEAM = HH48VMRLAB;
```

Comment:
Is this your development team?

Reply:
Yes, I changed it to install to the device. I'll remove it.